Show multiple charts with common legend

Bluesmile82 opened the pull request:
<img width="1201" alt="image" src="https://github.com/Vizzuality/laws_and_pathways/assets/9701591/10892e54-06bd-4dae-93da-fca490d77b1c">

Multiple charts are shown on the Carbon performance emission graphs per individual sectors

The shared legend is shown at the top.

https://vizzuality.atlassian.net/browse/LSE-84?atlOrigin=eyJpIjoiZTY5NTc3OThmNTkxNDkyZTg3MmNiYzAzZTAwZmE1ZjAiLCJwIjoiaiJ9

martintomas reviewed (changes requested):
I will not comment code, because I am not expert at this :). Anyways, I have tested it on my local machine and I think it looks overal good, just a few points:

- not really sure if we need legend per every graph, but I think this is up to Teona to decide
- when you resize screen, button called `See full sector chart` pops up. We need to get rid of it for Banks
<img width="755" alt="Screenshot 2023-07-04 at 9 38 04" src="https://github.com/Vizzuality/laws_and_pathways/assets/20660167/23875951-fca2-4300-a0a4-26b4d2d72cb6">

- When I open console, I see couple errors related to missing `unit` property. We should probably get rid of this error. Also some sectors don't have unit, but some of them have (for example unit of Steel sector is `Carbon intensity (tonnes of CO2 per tonne of steel)`). If possible, we should show such information somewhere at graph.

<img width="313" alt="Screenshot 2023-07-04 at 9 37 39" src="https://github.com/Vizzuality/laws_and_pathways/assets/20660167/1cacfb73-569b-4d4e-812d-508bfed64282">

Bluesmile82 commented:
Lots of thanks for the review. I added the unit and removed the sectors for this case
<img width="524" alt="image" src="https://github.com/Vizzuality/laws_and_pathways/assets/9701591/daf03bcb-f0de-4b71-b2b6-cf3a2d4685b5">

martintomas reviewed (approved):
👍
